package.json

```diff
@@ -10,7 +10,7 @@
     "test:legacy": "mocha test",
     "start": "node dist/index.js",
     "cloneEnv": "node dist/scripts/cloneEnv.js --clone",
-    "postinstall": "npm run db:migrate",
+    "postinstall": "npm run db:migrate && npm run build:src && npm run cloneEnv",
```

Comment:
Please add `concurrently` to the project, and use that to run multiple npm commands instead of the npm native way. @DarkStoorM